common/prometheus-server/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 7.0.11
+
+* Exclude `job=prometheus-vmware` from PrometheusMultipleTargetScrapes alert
```

Comment:
```suggestion
* Exclude `job=prometheus-vmware` from PrometheusMultipleTargetScrapes alert because this target is scraped with multiple jobs to the same target intentionally. Every scrape just replies a single metric.
```